CHANGELOG.md

```diff
@@ -0,0 +1,86 @@
+# GitPoint ChangeLog
+
+## master
+
+* Fixed crash in CodeLine component (#483)
+* Fixed locale detection (#479)
+* Added esperanto translations (#471)
+* Added polish translations (#466)
+* Fixed crash if markdown contained an anchor (#465)
+* Fixed issues links parsing (#463)
+* Added German translations (#458)
+* Added Galician translations (#452)
+* Fixed animated gif display on Android (#446)
+* Added portuguese translations (#440)
+* Fixed README file display on Android (#427)
+* Added check for PR mergeable status and disabled button accordingly (#425)
+* Added "Open in browser" functionality in various screens (#424)
+* Added spanish translations (#282)
+
+## Version 1.3.0 (2017-10-04)
```

Comment:
Will there be no releases of bug fixes? Version 1.3.1 is missing.

Reply:
It's not out yet as far as I understand

Reply:
![image](https://user-images.githubusercontent.com/4408379/31729110-c0c73348-b436-11e7-9eb1-b1511fffd0d4.png)

Reply:
yes, but it wasn't released yet. @housseindjirdeh mentioned on gitter that he'll be releasing it yesterday. Not sure if it was done, and what was included.

Reply:
No, this release with bug fixes has already been released (check Google Play), @housseindjirdeh meant a new release - 1.3.2.

Reply:
... but he did not update `package.json`, I wrote about it, but the release took place.

Reply:
Fixed, thanks a lot for the clarifications!